### PR TITLE
Update schema

### PR DESCRIPTION
**Pull request by cthoyt**
This PR does the following:

1. Require `preferredPrefix` annotation for active ontologies. It's already the case that all active ontologies contain an explicit preferred prefix, except SIBO, which I added
2. Set `description` level to "error". This seems obvious - all ontologies must have a description.
3. Remove several singleton fields. None should be controversial, I've put comments in the PR to note that some links were dead/content irrelevant/innapropriate
4. Remove unused fields from the schema

This PR only touched fields that were only used once. There are still a few used at most 3 times:

| key               | ontologies                                                                                                                                             |
|-------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
| createdWith       | [ms](https://obofoundry.org/ontologies/ms), [xlmod](https://obofoundry.org/ontologies/xlmod), [ncbitaxon](https://obofoundry.org/ontologies/ncbitaxon) |
| biosharing        | [uberon](https://obofoundry.org/ontologies/uberon), [miapa](https://obofoundry.org/ontologies/miapa)                                                   |
| releases          | [labo](https://obofoundry.org/ontologies/labo), [uberon](https://obofoundry.org/ontologies/uberon), [miapa](https://obofoundry.org/ontologies/miapa)   |
| documentation     | [ro](https://obofoundry.org/ontologies/ro), [pr](https://obofoundry.org/ontologies/pr), [ero](https://obofoundry.org/ontologies/ero)                   |
| alternativePrefix | [chebi](https://obofoundry.org/ontologies/chebi), [hp](https://obofoundry.org/ontologies/hp)                                                           |
| redirects         | [vto](https://obofoundry.org/ontologies/vto), [uberon](https://obofoundry.org/ontologies/uberon)                                                       |
| facebook          | [go](https://obofoundry.org/ontologies/go), [doid](https://obofoundry.org/ontologies/doid)                                                             |
| funded_by         | [nomen](https://obofoundry.org/ontologies/nomen), [eco](https://obofoundry.org/ontologies/eco), [uberon](https://obofoundry.org/ontologies/uberon)     |
| wasDerivedFrom    | [ncbitaxon](https://obofoundry.org/ontologies/ncbitaxon)                                                                                               |
| source            | [ncbitaxon](https://obofoundry.org/ontologies/ncbitaxon)                                                                                               |
| type              | [nomen](https://obofoundry.org/ontologies/nomen), [uberon](https://obofoundry.org/ontologies/uberon)                                                   |
| exampleClass      | [uberon](https://obofoundry.org/ontologies/uberon)                                                                                                     |
| slack             | [uberon](https://obofoundry.org/ontologies/uberon)                                                                                                     |
| wikidata_template | [uberon](https://obofoundry.org/ontologies/uberon)                                                                                                     |

### CODE REVIEW

File: ontology/doid.md [hunk 1]
```diff
@@ -11,7 +11,6 @@ twitter: diseaseontology
 facebook: https://www.facebook.com/diseaseontology
 domain: disease
 homepage: http://www.disease-ontology.org
-DO wiki: http://diseaseontology.sourceforge.net/
```
Comment (cthoyt) [inline ontology/doid.md:15]: this link is dead

File: ontology/emap.md [hunk 2]
```diff
@@ -20,9 +20,6 @@ build:
   source_url: ftp://ftp.hgu.mrc.ac.uk/pub/MouseAtlas/Anatomy/EMAP_combined.obo
   insert_ontology_id: true
   method: obo2owl
-used_by:
```
Comment (cthoyt) [inline ontology/emap.md:24]: this doesn't seem relevant / fit with the more widely used "usage" property

File: ontology/htn.md [hunk 3]
```diff
@@ -12,7 +12,6 @@ license:
   label: CC BY 4.0
   url: http://creativecommons.org/licenses/by/4.0/
 tracker: https://github.com/aellenhicks/htn_owl/issues
-source_url: https://raw.githubusercontent.com/aellenhicks/htn_owl/master/htn.owl
```
Comment (cthoyt) [inline ontology/htn.md:16]: maybe use `products` instead?

File: ontology/miro.md [hunk 4]
```diff
@@ -14,7 +14,6 @@ taxon:
   label: Anopheles
 title: Mosquito insecticide resistance
 activity_status: inactive
-validate: false
```
Comment (cthoyt) [inline ontology/miro.md:17]: This ontology is already marked as inactive, so this didn't seem necessary

File: ontology/ro.md [hunk 5]
```diff
@@ -8,7 +8,6 @@ build:
   method: vcs
   infallible: 1
 canonical: ro.owl
-aberowl_id: relo
```
Comment (cthoyt) [inline ontology/ro.md:11]: http://aber-owl.net/ontology/RO seems to work properly

File: ontology/htn.md (hunk 3 above)
Comment (matentzn) [reply ontology/htn.md:16]: No, you are absolutely right, this does not belong here - this goes into the purl config, where it already is.

File: ontology/ro.md (hunk 5 above)
Comment (matentzn) [reply ontology/ro.md:11]: Absolutely, thank you for checking!

File: ontology/doid.md (hunk 1 above)
Comment (matentzn) [reply ontology/doid.md:15]: Absolutely, let us remove this!

**Comment by matentzn** [comment]
This PR is _awesome_, and there wont be any argument here as you are not adding, but simplifying the metadata! THANK YOU!

Right now the QC is failing because some record still seems to have a `ontology_purl`? Maybe I read it wrong.

For your table above, I would remove:

- createdWith | ms, xlmod, ncbitaxon (this needs to either be done properly or not at all. Too much overhead for us).
- biosharing | uberon, miapa
- alternativePrefix | chebi, hp (this is so confusing, bioregistry material)
- redirects | vto, uberon (purvue of purl server, does not belong here and causes overhead/redundancy)
- source | ncbitaxon (confusing, and IMO redundant with "wasDerivedFrom"
- type | nomen, uberon (REMOVE)
- exampleClass | uberon (Redundant with PURL config)
- facebook | go, doid (don't care about this, but seems to be a remnant of time where facebook still was something)

Let us keep for now:

- releases | labo, uberon, miapa (this gives an opportunity to link to a webpage that gives access to all releases, maybe encourage more widely?)
- documentation | ro, pr, ero (definitely keep and roll out widely)
- wasDerivedFrom | ncbitaxon (this should be rolled out more widely)
- funded_by | nomen, eco, uberon (would be cool if we could standardise this. Can we refer to a grant using an identifier?)
- wikidata_template | uberon (not sure what this is, but maybe it makes sense)
- slack | uberon (definitely keep. Would be nice to find a way to link to a workspace to request to join, not sure if that is possible on free account though)

**Comment by cthoyt** [comment]
I made all suggested changes. It turns out the `ontology_purl` is automatically added by the build process, so as an alternative, I've upgraded this to be a "required" field. I'm not a big fan of the opaqueness of the current build process, but this is a job for a future PR.

**Comment by cthoyt** [comment]
Btw, here's the same table but with a cutoff of 15:

Fields used at most 15 times:
| key                | ontologies                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
|--------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
| label              | [ms](https://obofoundry.org/ontologies/ms), [xlmod](https://obofoundry.org/ontologies/xlmod), [epio](https://obofoundry.org/ontologies/epio), [one](https://obofoundry.org/ontologies/one), [ons](https://obofoundry.org/ontologies/ons), [cl](https://obofoundry.org/ontologies/cl), [uberon](https://obofoundry.org/ontologies/uberon), [nomen](https://obofoundry.org/ontologies/nomen), [go](https://obofoundry.org/ontologies/go), [mondo](https://obofoundry.org/ontologies/mondo)                                                                                                                                                   |
| integration_server | [ms](https://obofoundry.org/ontologies/ms), [obi](https://obofoundry.org/ontologies/obi), [xlmod](https://obofoundry.org/ontologies/xlmod), [cl](https://obofoundry.org/ontologies/cl), [go](https://obofoundry.org/ontologies/go)                                                                                                                                                                                                                                                                                                                                                                                                         |
| releases           | [labo](https://obofoundry.org/ontologies/labo), [miapa](https://obofoundry.org/ontologies/miapa), [uberon](https://obofoundry.org/ontologies/uberon)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
| documentation      | [ero](https://obofoundry.org/ontologies/ero), [pr](https://obofoundry.org/ontologies/pr), [ro](https://obofoundry.org/ontologies/ro)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
| canonical          | [cl](https://obofoundry.org/ontologies/cl), [uberon](https://obofoundry.org/ontologies/uberon), [nomen](https://obofoundry.org/ontologies/nomen), [ro](https://obofoundry.org/ontologies/ro), [mondo](https://obofoundry.org/ontologies/mondo)                                                                                                                                                                                                                                                                                                                                                                                             |
| twitter            | [doid](https://obofoundry.org/ontologies/doid), [chebi](https://obofoundry.org/ontologies/chebi), [ddpheno](https://obofoundry.org/ontologies/ddpheno), [hp](https://obofoundry.org/ontologies/hp), [go](https://obofoundry.org/ontologies/go), [uberon](https://obofoundry.org/ontologies/uberon), [disdriv](https://obofoundry.org/ontologies/disdriv), [ddanat](https://obofoundry.org/ontologies/ddanat), [po](https://obofoundry.org/ontologies/po)                                                                                                                                                                                   |
| depicted_by        | [stato](https://obofoundry.org/ontologies/stato), [obi](https://obofoundry.org/ontologies/obi), [chebi](https://obofoundry.org/ontologies/chebi), [mondo](https://obofoundry.org/ontologies/mondo), [so](https://obofoundry.org/ontologies/so), [iao](https://obofoundry.org/ontologies/iao), [uberon](https://obofoundry.org/ontologies/uberon), [fypo](https://obofoundry.org/ontologies/fypo), [eupath](https://obofoundry.org/ontologies/eupath), [ogms](https://obofoundry.org/ontologies/ogms), [go](https://obofoundry.org/ontologies/go), [pr](https://obofoundry.org/ontologies/pr), [bfo](https://obofoundry.org/ontologies/bfo) |
| in_foundry         | [uo](https://obofoundry.org/ontologies/uo), [bcgo](https://obofoundry.org/ontologies/bcgo), [flu](https://obofoundry.org/ontologies/flu), [fix](https://obofoundry.org/ontologies/fix), [bco](https://obofoundry.org/ontologies/bco), [obcs](https://obofoundry.org/ontologies/obcs), [zfs](https://obofoundry.org/ontologies/zfs), [epo](https://obofoundry.org/ontologies/epo), [lipro](https://obofoundry.org/ontologies/lipro), [stato](https://obofoundry.org/ontologies/stato), [omit](https://obofoundry.org/ontologies/omit)                                                                                                       |
| review             | [doid](https://obofoundry.org/ontologies/doid), [pato](https://obofoundry.org/ontologies/pato), [obi](https://obofoundry.org/ontologies/obi), [chebi](https://obofoundry.org/ontologies/chebi), [xao](https://obofoundry.org/ontologies/xao), [bfo](https://obofoundry.org/ontologies/bfo), [zfa](https://obofoundry.org/ontologies/zfa), [go](https://obofoundry.org/ontologies/go), [pr](https://obofoundry.org/ontologies/pr), [po](https://obofoundry.org/ontologies/po)                                                                                                                                                               |
| in_foundry_order   | [doid](https://obofoundry.org/ontologies/doid), [pato](https://obofoundry.org/ontologies/pato), [obi](https://obofoundry.org/ontologies/obi), [chebi](https://obofoundry.org/ontologies/chebi), [xao](https://obofoundry.org/ontologies/xao), [bfo](https://obofoundry.org/ontologies/bfo), [zfa](https://obofoundry.org/ontologies/zfa), [go](https://obofoundry.org/ontologies/go), [pr](https://obofoundry.org/ontologies/pr), [po](https://obofoundry.org/ontologies/po)                                                                                                                                                               |
| funded_by          | [nomen](https://obofoundry.org/ontologies/nomen), [uberon](https://obofoundry.org/ontologies/uberon), [eco](https://obofoundry.org/ontologies/eco)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
| wasDerivedFrom     | [ncbitaxon](https://obofoundry.org/ontologies/ncbitaxon)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
| slack              | [uberon](https://obofoundry.org/ontologies/uberon)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |
| wikidata_template  | [uberon](https://obofoundry.org/ontologies/uberon)                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         |

**Review by matentzn — APPROVED** [review]
Loving it.

File: ontology/pr.md [hunk 6]
```diff
@@ -41,8 +41,6 @@ tracker: https://github.com/PROconsortium/PRoteinOntology/issues
 publications:
   - id: https://proconsortium.org/pro_dsmnt.shtml#publication
     title: "Publications & Dissemination"
-development:
-  id_policy: https://pir17.georgetown.edu/confluence/display/PROWIKI/PRO+PURLs
```
[the remark this answers is not in the document]
Comment (cthoyt) [reply ontology/pr.md:46]: tbh I don't actually subscribe to the "Termina is purgatory" theory of Majora's Mask

[the remark this answers is not in the document]
Comment (matentzn) [reply ontology/pr.md:46]: What??? Hahaha dont get what this means but the picture looks like me after 10 minutes treadmill

**Pull request merged**